packages/firestore/e2e/firestore.e2e.js

```diff
@@ -364,4 +364,42 @@ describe('firestore()', () => {
       }
     });
   });
+
+  describe('wait for pending writes', () => {
+    it('waits for pending writes', async () => {
+
+      const waitForPromiseMs = 500;
+      const testTimeoutMs = 10000;
+
+      await firebase.firestore().disableNetwork();
+
+      //set up a pending write
+
+      const db = firebase.firestore();
+      const id = 'foobar';
+      const ref = db.doc(`v6/${id}`);
+      ref.set({ foo: 'bar' });
+
+      //waitForPendingWrites should never resolve, but unfortunately we can only 
+      //test that this is not returning within X ms
```

Comment:
I think that's totally fair, testing is not perfect but just that it hangs is a pretty good indication it's working